vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data.cc

```diff
@@ -137,6 +138,61 @@ double operator*(const VectorData& lhs, const VectorData& rhs) {
   return dot_product;
 }
 
+void VectorData::VectorAddElementWise(const VectorData& v_add) {
```

Comment:
might be worth adding an `InPlace` specifier to these functions that add the argument vector to the caller

Reply:
didn't we decide to keep it as is since the signature implies in-place?

Reply:
I think that with the current naming when the function is called there'd still be ambiguity on whether a new value is returned or the addition happens in place

Reply:
@tmancey wdyt?

Reply:
reading `VectorAddElementWise` does not tell me what is happening. What is Wise?

Reply:
Wise as in element-wise (see an example [here](https://numpy.org/doc/stable/reference/generated/numpy.multiply.html)). The distinction is mainly useful for multiplication, rather than addition, but it will allow consistency in naming convention as we add functions such as `VectorMultiplyElementWise `

Reply:
Based on the example how about AddVectorElementWise and MultiplyVectorElementWise?

Reply:
Updated to `AddElementWise`. Will push with next commit

Reply:
nit: @ptjames looking at tensor flow and other areas of Chromium which use Elementwise the naming convention is `AddElementwise` (lowercase w), i.e. see `src/third_party/tflite/src/tensorflow/lite/kernels/internal/optimized/optimized_ops.h`